gammapy/cube/simulate.py

```diff
@@ -0,0 +1,82 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Simulate observations"""
+import astropy.units as u
+from ..cube import MapDataset, PSFKernel
+from ..cube import make_map_exposure_true_energy, make_map_background_irf
+from ..maps import WcsNDMap
+from ..cube.models import BackgroundModel
+from ..utils.random import get_random_state
+
+__all__ = ["simulate_3d"]
+
+
+def simulate_3d(
```

Comment:
Maybe rename to `simulate_map_dataset`? I think it describes better what it does...